openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataInsightAlert.spec.js

```diff
@@ -0,0 +1,209 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import {
+  descriptionBox,
+  interceptURL,
+  verifyResponseStatusCode,
+} from '../../common/common';
+
+const dataInsightReport = {
+  triggerType: 'Scheduled',
+  scheduleInfoType: 'Weekly',
+  name: 'DataInsightReport',
+  description:
+    'Data Insight Report send to the admin (organization level) and teams (team level) at given interval.',
+  updatedDescription: 'Updated Description',
+};
+
+describe('Data Insight Alert', () => {
+  beforeEach(() => {
+    cy.login();
+
+    cy.get('[data-testid="appbar-item-settings"]')
+      .should('exist')
+      .and('be.visible')
+      .click();
+
+    interceptURL(
+      'GET',
+      'api/v1/events/subscriptions/name/DataInsightReport?include=all',
+      'dataInsightReport'
+    );
+
+    cy.get('[data-testid="global-setting-left-panel"]')
+      .contains('Data Insight Report')
+      .scrollIntoView()
+      .should('be.visible')
+      .and('exist')
+      .click();
+
+    verifyResponseStatusCode('@dataInsightReport', 200);
+  });
+
+  it('Should have default alert with proper data', () => {
+    cy.get('.sub-heading')
+      .should('be.visible')
+      .contains(dataInsightReport.description);
```

Comment:
heading selection should be selected with data-testid, if css change in future it can lead to failure of this test